common/prettify.pm

```diff
@@ -655,6 +655,14 @@ sub Footer ()
           print $out "<failure>\n",
               $indent x 3, "<![CDATA[$error]]></failure><system-out>\n",
               $indent x 3, "<![CDATA[$test->{OUT}]]></system-out></testcase>\n";
+
+          my $github_actions = main::GetVariable ('GITHUB_ACTIONS');
```

Comment:
What about using the `$GITHUB_ACTIONS` environment variable instead? That way it's automatic. Or better yet it's true if the environment variable is defined unless the autobuild variable is set to 0.